flink-sql-connector-oceanbase-cdc/pom.xml

```diff
@@ -64,12 +64,9 @@ under the License.
                                     <include>commons-codec:commons-codec</include>
                                     <include>org.lz4:lz4-java</include>
                                     <include>org.apache.avro:avro</include>
-                                    <include>org.apache.commons:*</include>
+                                    <include>org.apache.commons:commons-lang3</include>
                                     <include>org.apache.kafka:*</include>
                                     <include>com.fasterxml.*:*</include>
-                                    <include>com.google.guava:*</include>
-                                    <!--  Include fixed version 18.0-13.0 of flink shaded guava  -->
-                                    <include>org.apache.flink:flink-shaded-guava</include>
```

Comment:
This is for compatible with Flink 1.14 ? why we change this

Reply:
`com.google.guava:guava` and `org.apache.commons:commons-collections4` are removed in oblogclient 1.0.3 , they are not required now. https://github.com/oceanbase/oblogclient/pull/24

Reply:
but why you remove the flink shaeded guava?

Reply:
Sure, will do it in next week.